habitat/parser.py

```diff
@@ -85,6 +85,20 @@ def __init__(self, config):
         self.couch_server = couchdbkit.Server(config["couch_uri"])
         self.db = self.couch_server[config["couch_db"]]
 
+        # Grab the radiosonde override config.
+        self.rs_prefix = "RS_" # Default radiosonde callsign identifier.
+        self.rs_config = None
+        try:
+            self.rs_prefix = config["radiosonde_override_prefix"]
+            _rs_config = self.db[config["radiosonde_override"]]
+            self.rs_config = {'payload_configuration': _rs_config, 'id': config["radiosonde_override"]}
+            logging.debug("Loaded radiosonde override payload doc (%s)" % config["radiosonde_override"])
+        except Exception as e:
+            logging.debug("Could not load radiosonde override payload doc - %s" % str(e))
+
+
+            
+
```

Comment:
Style is for two blank lines here not four. You might find running `flake8` or `pep8` over the source will give some hints for formatting and many editors can be configured to do this automatically for all Python files.

Reply:
According to PEP E303 this should be one blank line... https://www.flake8rules.com/rules/E303.html